<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>500</width>
                <height>260</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>SITN 4D -Historisation d'une table</string>
        </property>
        <widget class="QComboBox" name="mapLayerComboBox">
            <property name="enabled">
                <bool>false</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>40</y>
                    <width>271</width>
                    <height>27</height>
                </rect>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
            <property name="geometry">
                <rect>
                    <x>150</x>
                    <y>190</y>
                    <width>211</width>
                    <height>31</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <pointsize>12</pointsize>
                </font>
            </property>
            <property name="styleSheet">
                <string notr="true">color:white;
background-color:#1E90FF;</string>
            </property>
            <property name="text">
                <string>✔️ Valider</string>
            </property>
            <property name="checkable">
                <bool>false</bool>
            </property>
        </widget>
        <widget class="QComboBox" name="libelleComboBox">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>90</y>
                    <width>271</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QComboBox" name="idComboBox">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>140</y>
                    <width>271</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QLabel" name="label">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>50</y>
                    <width>91</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Table</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_2">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>90</y>
                    <width>91</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Champ libellé</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_3">
            <property name="geometry">
                <rect>
                    <x>50</x>
                    <y>140</y>
                    <width>91</width>
                    <height>16</height>
                </rect>
            </property>
            <property name="text">
                <string>Champ ID</string>
            </property>
        </widget>
    </widget>
    <customwidgets>
        <customwidget>
            <class>QgsMapLayerComboBox</class>
            <extends>QComboBox</extends>
            <header>qgsmaplayercombobox.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
